openapi/wicketd.json

```diff
@@ -3582,7 +3582,7 @@
         ]
       },
       "RotImageError": {
-        "description": "RotImageError\n\n<details><summary>JSON schema</summary>\n\n```json { \"type\": \"string\", \"enum\": [ \"unchecked\", \"first_page_erased\", \"partially_programmed\", \"invalid_length\", \"header_not_programmed\", \"bootloader_too_small\", \"bad_magic\", \"header_image_size\", \"unaligned_length\", \"unsupported_type\", \"reset_vector_not_thumb2\", \"reset_vector\", \"signature\" ] } ``` </details>",
+        "description": "`RotImageError`\n\n<details><summary>JSON schema</summary>\n\n```json { \"type\": \"string\", \"enum\": [ \"unchecked\", \"first_page_erased\", \"partially_programmed\", \"invalid_length\", \"header_not_programmed\", \"bootloader_too_small\", \"bad_magic\", \"header_image_size\", \"unaligned_length\", \"unsupported_type\", \"reset_vector_not_thumb2\", \"reset_vector\", \"signature\" ] } ``` </details>",
```

Comment:
Seems like the only change to the OpenAPI schemas is the addition of backticks around the schema name in each case. Sure, whatever!

Reply:
yeah, that's in https://github.com/oxidecomputer/omicron/pull/8086 too, just fallout from the Progenitor version bump. ... i was not sure why only these items were affected but  it looks like this is a default description if the item doesn't have one, and everything else has doc comments.

this includes the Progenitor bump in 8086 only because Propolis is now at Progenitor 0.10.0. if you want to go sign off on 8086 though, it'll make the diff here much much more clear (and tbh the two separate piece being in two different commits makes a lot more sense to me)